extension.js

```diff
@@ -100,14 +100,30 @@ const stringFunction = async (commandName) => {
       builder.replace(selectionMap[replaced], replaced);
     });
   });
+
+  context.globalState.update('lastAction', commandName);
```

Comment:
globalState is for all of vscode. I'd recommend prefixing "lastAction" with the extension name or something to make it more unique to prevent possible conflicts.